assignments/prototypes.js

```diff
@@ -8,40 +8,64 @@
   Each constructor function has unique properites and methods that are defined in their block comments below:
 */
   
-/*
-  === GameObject ===
-  * createdAt
-  * dimensions
-  * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
-*/
 
-/*
-  === CharacterStats ===
-  * hp
-  * name
-  * takeDamage() // prototype method -> returns the string '<object name> took damage.'
-  * should inherit destroy() from GameObject's prototype
-*/
+  // === GameObject ===
+  // * createdAt
+  // * dimensions
+  // * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
+  
+  function GameObject(gameAttr) {
+    this.createdAt = gameAttr.createAt;
+    this.dimensions = gameAttr.dimensions;
+    this.destroy = function (){
```

Comment:
This is one way to add the destroy method to GameObject. Best practices is to add the method on after the constructor function is defined. delete this method off the constructor function and bind it to GameObject using prototype.